addons/common/init_perFrameHandler.sqf

```diff
@@ -14,11 +13,34 @@ GVAR(nextFrameBufferA) = [];
 GVAR(nextFrameBufferB) = [];
 GVAR(waitUntilAndExecArray) = [];
 
+GVAR(missionTimeSynchronized) = false;
+GVAR(missionTimePrecise) = 0;
+GVAR(missionTimeThousands) = 0;
+CBA_missionTime = 0;
+CBA_missionTimeStr = "0000.000000";
+
 // per frame handler system
 [QFUNC(onFrame), {
     SCRIPT(onFrame);
     private _tickTime = diag_tickTime;
-    call FUNC(missionTimePFH);
+
+    // Update CBA_missionTime when synchronized, time has started, and game is not paused.
+    if ([GVAR(missionTimeSynchronized), time > 0, isGamePaused] isEqualTo [true, true, false]) then {
```

Comment:
I think this is better written as:
```suggestion
    if (GVAR(missionTimeSynchronized) && {time > 0 && {!isGamePaused}}) then {
```

Reply:
Isn't that slower though? That shit runs every frame.

Reply:
It is slightly faster.
```
Result:
0.0028 ms

Cycles:
10000/10000

Code:
[cba_common_missionTimeSynchronized, time > 0, isGamePaused] isEqualTo [true, true, false]
```
```
Result:
0.0021 ms

Cycles:
10000/10000

Code:
cba_common_missionTimeSynchronized && {time > 0 && {!isGamePaused}}
```

Reply:
What was `cba_common_missionTimeSynchronized` in your example?` It will be `true` all mission long.

Reply:
It was `true`; the overall result for both statements was also `true`.

Reply:
hows the speed without the lazy eval?
creating array, calling "true" command 3 times, creating another array, calling isEqualTo command surely is a bit expensive.
But lazy eval is also expensive but in this case as commy said, we don't expect neither missionTimeSynchronized or the time check to ever be false, so why lazy eval?

Reply:
Yeah, no lazy eval is the fastest.

```
Result:
0.0017 ms

Cycles:
10000/10000

Code:
cba_common_missionTimeSynchronized && time > 0 && !isGamePaused
```

Updated the suggestion.